Library/Homebrew/extend/os/mac/ENV/super.rb

```diff
@@ -0,0 +1,104 @@
+    end
+
+    paths << MacOS::X11.bin.to_s if x11?
+    paths
+  end
+
+  # @private
+  def homebrew_extra_pkg_config_paths
+    paths = ["#{HOMEBREW_ENV_PATH}/pkgconfig/#{MacOS.version}"]
+    paths << "#{MacOS::X11.lib}/pkgconfig" << "#{MacOS::X11.share}/pkgconfig" if x11?
+    paths
+  end
+
+  def homebrew_extra_aclocal_paths
+    paths = []
+    paths << "#{MacOS::X11.share}/aclocal" if x11?
+    paths
+  end
+
+  def homebrew_extra_isystem_paths
+    paths = []
+    paths << "#{effective_sysroot}/usr/include/libxml2" unless deps.any? { |d| d.name == "libxml2" }
+    paths << "#{effective_sysroot}/usr/include/apache2" if MacOS::Xcode.without_clt?
+    paths << MacOS::X11.include.to_s << "#{MacOS::X11.include}/freetype2" if x11?
+    paths << "#{effective_sysroot}/System/Library/Frameworks/OpenGL.framework/Versions/Current/Headers"
+    paths
+  end
+
+  def homebrew_extra_library_paths
+    paths = []
+    paths << MacOS::X11.lib.to_s if x11?
+    paths << "#{effective_sysroot}/System/Library/Frameworks/OpenGL.framework/Versions/Current/Libraries"
+    paths
+  end
+
+  def homebrew_extra_cmake_include_paths
+    paths = []
+    paths << "#{effective_sysroot}/usr/include/apache2" if MacOS::Xcode.without_clt?
+    paths << MacOS::X11.include.to_s << "#{MacOS::X11.include}/freetype2" if x11?
+    paths << "#{effective_sysroot}/System/Library/Frameworks/OpenGL.framework/Versions/Current/Headers"
+    paths
+  end
+
+  def homebrew_extra_cmake_library_paths
+    paths = []
+    paths << MacOS::X11.lib.to_s if x11?
+    paths << "#{effective_sysroot}/System/Library/Frameworks/OpenGL.framework/Versions/Current/Libraries"
+    paths
+  end
+
+  def homebrew_extra_cmake_frameworks_paths
+    paths = []
+    paths << "#{effective_sysroot}/System/Library/Frameworks" if MacOS::Xcode.without_clt?
+    paths
+  end
+
+  def determine_cccfg
+    s = ""
+    # Fix issue with sed barfing on unicode characters on Mountain Lion
+    s << "s" if MacOS.version >= :mountain_lion
+    # Fix issue with >= 10.8 apr-1-config having broken paths
+    s << "a" if MacOS.version >= :mountain_lion
+    s
+  end
+
+  # @private
+  def setup_build_environment(formula = nil)
+    generic_setup_build_environment(formula)
+    self["HOMEBREW_SDKROOT"] = effective_sysroot
+
+    if MacOS::Xcode.without_clt? || (MacOS::Xcode.installed? && MacOS::Xcode.version.to_i >= 7)
+      self["MACOSX_DEPLOYMENT_TARGET"] = MacOS.version.to_s
+      self["SDKROOT"] = MacOS.sdk_path
+    end
+
+    # On 10.9, the tools in /usr/bin proxy to the active developer directory.
+    # This means we can use them for any combination of CLT and Xcode.
+    self["HOMEBREW_PREFER_CLT_PROXIES"] = "1" if MacOS.version >= "10.9"
+  end
+
+  def x11_if_installed
+    ENV.x11 = MacOS::X11.installed?
+  end
+
+  # These methods are no longer necessary under superenv, but are needed to
+  # maintain an interface compatible with stdenv.
+  noops.concat %w[macosxsdk remove_macosxsdk]
```

Comment:
This code does nothing, because this `noops` array has already been processed at the end of `extend/ENV/super.rb`. I'd suggest to just make this:

```ruby
alias_method :macosxsdk, :noop
alias_method :remove_macosxsdk, :noop
```